modules/ROOT/pages/salesforce/salesforce-mktg-connector-reference.adoc

```diff
@@ -1,661 +1,778 @@
-| Read Timeout a| Number |  +++Specifies the amount of time in milliseconds that the consumer waits for a response before it times out. Default value is 0, which means the value used by Fuel SDK is used.+++ |  |
-| Connection Timeout a| Number |  +++Specifies the amount of time in milliseconds that the consumer attempts to establish a connection before it times out. Default value is 0, which means the value used by Fuel SDK is used.+++ |  |
-| Reconnection a| <<Reconnection>> |  +++When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy.+++ |  |
+
+[%header%autowidth.spread]
 |===
+| Connection Timeout | Type | Description | Default Value | Required
+| Service URL a| String |  The base URL of the service on Salesforce Marketing Cloud to be accessed from the connector. From this base URL all other endpoints derive. |  | x
+| Connection Timeout a| Number |  The socket connection timeout value. This attribute works in tandem with the Connection Timeout Unit. |  |
+| Connection Timeout Unit a| Enumeration, one of:
 
-== Operations
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the Connection Timeout. |  SECONDS |
+| Proxy configuration a| <<ProxyConfiguration>> |  Proxy configuration for the connector. |  |
+| WSDL file URL a| String |  The URL of the WSDL file used by this connector. When no value is set, this parameter defaults to the Service URL parameter host, but it appends /etframework.wsdl at its end. |  |
+| TLS configuration a| <<Tls>> |  Protocol to use for communication. Valid values are HTTP and HTTPS. Default value is HTTP. When using HTTPS, the HTTP communication is secured using TLS / SSL. If HTTPS was configured as the protocol, the user needs to configure at least the keystore in the tls:context child element of this listener-config. |  |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+| Client Id a| String |  The OAuth client ID as registered with the service provider. |  | x
+| Client Secret a| String |  The OAuth client secret as registered with the service provider. |  | x
+| Token Url a| String |  The service provider's token endpoint URL. |  https://{domain}.auth.marketingcloudapis.com/v2/token |
+| Scopes a| String |  The OAuth scopes to be requested during the dance. If not provided, it defaults to those in the annotation. |  |
+| Object Store a| String |  A reference to the object store that should be used to store each resource owner id's data. If not specified, runtime will automatically provision the default one. |  |
+|===
+
+== Supported Operations
```

Comment:
Move the list that is under "Supported Operations" to under the next heading "Operations" and delete the "Supported Operations" header.